src/app/routes.tsx

```diff
@@ -117,10 +117,10 @@ const routes: IAppRoute[] = [
   {
     component: CreateCache,
     exact: true,
-    label: 'Caches',
+    label: 'Cache Wizard',
     path: '/container/:cmName/caches/create',
```

Comment:
"cmName" not fetching from Router.
<img src="https://user-images.githubusercontent.com/97534722/189580784-dba1da8c-bfa5-4541-a274-e02dbe430fe4.png" width="50%">

Reply:
I added a fix. We don't need the cache manager name in the url in that case, but other cases use it too. We need to do a proper solution to grab the cache manager information and pass it around. probably with state providers and hooks. 

For now I think we can use what I've done in the commit. Please check it and rebase/ merge / format the stuff properly. Probably we need a "Loading"  generic component too that could be re used around as the error one.